app.js

```diff
@@ -1,2 +1,31 @@
-const BOARD = document.getElementById('letters');
+const gameWrapper = document.querySelector('.game-wrapper');
+gameSetup();
 
+function gameSetup() {
+  for (i = 0; i < 26; i++) {
```

Comment:
I would do slightly another approach. 
You can create an array of letters in the order as you are doing here using charCode method. 

And then use a `forEach`. We don't like for loop, also it is very cryptic now.